src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -180,4 +213,15 @@ public void writeThrough(Conversation conversation) {
     conversationEntity.setProperty("creation_time", conversation.getCreationTime().toString());
     datastore.put(conversationEntity);
   }
+
+  /** Write a Hashtag object to the Datastore service. */
+  public void writeThrough(Hashtag hashtag) {
```

Comment:
Good, you see how you saved all of these fields, you need to retrieve all of them during the Loading.